commnad_line_rock_paper_scissors.py

```diff
@@ -6,6 +6,27 @@
 from rock_paper_scissors.game import Game
 
 
+def human_input_to_move(game, human):
+    choices = ', or '.join(
+        [
+            ', '.join(game.valid_choices[:-1]),
+            game.valid_choices[-1]
+        ]
+    )
+
+    try:
+        human.choice = game.convert_input_to_move(input('Choose {}: '.format(choices)))
```

Comment:
I may be misunderstanding, but it's correct to have the `return` statement in both places as you do now. Since the purpose of this function is to:
- Prompt a user for input
- Return validated input, or
- Give an error message an rerun the function so the user is prompted again

then having two return statements is correct.

Let me know if I'm missing something, though, and I can try to clear things up.